### PR TITLE
Resolving Fetch issues

### PR DESCRIPTION
**Pull request by emtwo**

### CODE REVIEW

**Comment by pdehaan** [comment]
This is choking on Travis-CI (due to webpack.BannerPlugin &mdash; although the usage looks correct based on my 2 second ~~Googling~~ Yahoo!ing of [the Webpack BannerPlugin docs](https://webpack.js.org/plugins/banner-plugin/)).

I'll see if I can get it working locally:

```sh
WARNING in ping-centre.min.js from UglifyJs
Condition always true [./~/joi-browser/dist/joi-browser.js:2,0]
Dropping unreachable code [./~/joi-browser/dist/joi-browser.js:4,0]
Dropping unused variable internals [./~/joi-browser/dist/joi-browser.js:5383,0]
Dropping unused variable internals [./~/joi-browser/dist/joi-browser.js:5753,0]
Dropping unused variable internals [./~/joi-browser/dist/joi-browser.js:10023,0]
/home/travis/build/mozilla/ping-centre/node_modules/webpack/lib/BannerPlugin.js:9
	if(str.indexOf("\n") < 0) return "/*! " + str + " */";
	       ^
TypeError: str.indexOf is not a function
    at wrapComment (/home/travis/build/mozilla/ping-centre/node_modules/webpack/lib/BannerPlugin.js:9:9)
    at new BannerPlugin (/home/travis/build/mozilla/ping-centre/node_modules/webpack/lib/BannerPlugin.js:15:44)
    at Object.<anonymous> (/home/travis/build/mozilla/ping-centre/webpack.addon.config.js:22:5)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
```

**Comment by emtwo** [comment]
I made some changes that should make it a lot easier to use ping centre in a Firefox addon. Let me know what you think of this (next week when you're back)

File: README.md [hunk 1]
```diff
@@ -9,9 +9,27 @@ $ npm install ping-centre
...
+
+1. Run npm run bundle to output bundles to `dist/`
+2. Place the `ping-centre.addon.min.js` bundle in the addon directory
+3. Import as follows: `const PingCentre = require("./ping-centre.addon.min");`
 
+## Browser
+
+1. Run npm run bundle to output bundles to `dist/`
+2. Place the `ping-centre.min.js` bundle in your project directory
+3. Import as follows: `<script type="text/javascript" src="ding-centre.min.js" charset="utf-8"></script>`
```
Comment (pdehaan) [inline README.md]: typo: ding-centre

Comment (emtwo) [inline README.md]: Thanks lol

**Pull request closed without merging**